MeetupPlatformApi/Features/Meetups/GetMeetups/Filter/FilterSettings/MeetupsFilterSettings.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace MeetupPlatformApi.Features.Meetups.GetMeetups.Filter.FilterSettings;
+
+using System.ComponentModel.DataAnnotations;
+
+public class MeetupsFilterSettings
+{
+    /// <summary>
+    /// Number of page
+    /// </summary>
+    /// <example>2</example>
+    [Required]
+    [Range(1,int.MaxValue)]
+    public int PageNumber { get; set; }
+
+    /// <summary>
+    /// The number of meetups that will fit on one page
+    /// </summary>
+    /// <example>10</example>
+    [Required]
+    [Range(1, 100)]
+    public int PageSize { get; set; }
+
+    /// <summary>
+    /// Parameter by which to sort meetups
+    /// </summary>
+    /// <example>1</example>
+    [Required]
+    public SortOptions SortOption { get; set; }
```

Comment:
Right now user can only provide a number, but it would be really nice, if You've provided support for string values.

Reply:
Adding validation would also be very nice of You